package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mojaloop/ml-api-adapter",
-  "version": "5.1.2",
+  "version": "5.2.1",
```

Comment:
This needs to be v5.2.0. You may check the releases for a repo: https://github.com/mojaloop/ml-api-adapter/releases to help with this. (The last release for this service was v5.1.1)

Reply:
Updated to 5.2.0